### PR TITLE
[WIP] Get Frankfurt onruby ;)

### PR DESCRIPTION
**Pull request by klaustopher**
Heyo everyone,

after having taken over control of the Frankfurt Ruby Meetup, we are migrating it away from meetup.com and what other resource would be better to come here? :D

This is just a PR to open up the conversation, I just started with the raw fork.

Still some ToDo:

- [ ] Add the current logo and all descriptions
- [ ] Link ruby-frankfurt.de to heroku
- [ ] Set up a patreon donation

### CODE REVIEW

File: config/locales/de.label.yml [hunk 1]
```diff
@@ -111,3 +112,9 @@ de:
       :subtitle: Andalucia Ruby User Group
       :meta_desc: Andalucia Ruby User Group
       :custom_recurrence: letzter Dienstag des Monats um 19:30 Uhr
+    frankfurt:
+      :city: City
```
Comment (bogn83) [inline config/locales/de.label.yml]: These keys need to be updated I guess:
      name: Frankfurt.rb
      title: Frankfurt.rb
      subtitle: Ruby User Group Frankfurt
      meta_desc: Ruby / Rails User Group Frankfurt

**Comment by phoet** [comment]
👋 hi frankfurt!

yes, please try to add as much meta information as possible. throw away what you dont need.

i've already prepared cloudflare, so if you add those external nameservers:

```
dana.ns.cloudflare.com
will.ns.cloudflare.com
```

you would be good to go with SSL connections etc.

if you are planning to import any data i can help you get it imported into the database.

cheers! 🎉

**Comment by klaustopher** [comment]
I don't think external DNS will work since we also have email and stuff set up. Any A/CNAME/etc links don't work?

**Comment by phoet** [comment]
@klaustopher  cloudflare copied those

Screen Shot 2019-03-20 at 16 24 36

unfortunately, if you want to use SSL termination from cloudflare, you have to use external dns

**Comment by klaustopher** [comment]
Let's see if I want to trust you with my DNS 🤔 😁

**Comment by phoet** [comment]
@klaustopher that is an issue, for sure. the problem is though, that setting up SSL is tricky and without control over the process, i'm not willing to support it. the reasons are in this 2 year old issue https://github.com/phoet/on_ruby/issues/360

**Comment by klaustopher** [comment]
I was mostly kidding ;)

**Comment by phoet** [comment]
@klaustopher is there anything i can help you with?

**Comment by klaustopher** [comment]
Buy me some free time...

**Comment by phoet** [comment]
💸 there you go!

habt ihr keine hackdays in diesem ioki dingens?

**Comment by klaustopher** [comment]
Keinen zeitnahen ... Ich schau, dass ich mir am Wochenende was freischaufel

**Comment by phoet** [comment]
@klaustopher ich kann gerne dabei helfen hier noch etwas zu vervollständigen. musst mir nur verraten wo ich die aktuellsten infos zur UG finde.

hier?
https://www.meetup.com/de-DE/Frankfurt-Ruby-Meetup/
https://twitter.com/ruby_frankfurt

**Comment by klaustopher** [comment]
Ich komm einfach im Moment zu nichts, Logo in passende Form bringen und so ... Ich mach hier erst mal zu und mach demnächst nen neuen auf, wenn ich auch Zeit habe mich zu kümmern.

Sorry, und danke fürs Support anbieten.

**Comment by phoet** [comment]
Neben der Softwareentwicklung gebe ich auch Seminare in Zeitmanagement und Selbstorganisation 😋

**Pull request closed without merging**